docs/pages/upcoming-releases.mdx

```diff
@@ -108,8 +83,6 @@ The key deliverables for Teleport Cloud in the next quarter:
 
 | Week of            | Description                                                    |
 |--------------------|----------------------------------------------------------------|
-| August 28, 2024    | Teleport 16.1.8 will begin rollout on Cloud.                   |
-| August 28, 2024    | Teleport 16.1.8 agents will begin rollout to eligible tenants. |
 | September 02, 2024 | Teleport 16.2 will begin rollout on Cloud.                     |
 | September 02, 2024 | Teleport 16.2 agents will begin rollout to eligible tenants.   |
```

Comment:
```suggestion
| September 03, 2024 | Teleport 16.2 agents will begin rollout to eligible tenants.   |
```
Should we bump the agent update to be +1 from the Cloud rollout date?  Even though agents should be backwards compatible the typical workflow is for us to upgrade those after all tenants are upgraded.  The last being the 08 UTC window on the following day.  Upgrade example with windows: 16:00 UTC 09/02/24 -> 23:00 UTC 09/02/24 -> 08:00 UTC 09/03/24

If we go this route also update the 09/16 date

Reply:
Just a heads up, this is `Week of`.

Reply:
Ah I missed that thanks.